README.md

```diff
@@ -1,6 +1,106 @@
+  - Python: [sigstore-python](https://github.com/sigstore/sigstore-python)
+  - Ruby: [sigstore-ruby](https://github.com/sigstore/sigstore-ruby)
+  - Rust: [sigstore-rs](https://github.com/sigstore/sigstore-rs)
+[policy-controller](https://github.com/sigstore/policy-controller)
+
+## Get Involved
+
+(*You'll need to join [sigstore-dev@googlegroups.com](https://groups.google.com/g/sigstore-dev) for access to many of these (to prevent spam).*)
+
+We welcome contributions from all! Great ways to help include:
+
+- Use a Sigstore client and provide feedback (in the form of GitHub issues, chatter on Slack, etc.).
+- Contribute to any of the above projects: you can just jump in on GitHub (generally best to file issues, ask whether anybody is working on something, etc. before just firing off a PR; see the `CONTRIBUTORS.md` or `CONTRIBUTING.md` in the respective repository).
+- Say hi in Slack!
+- Join a meeting (open to all community members; see below).
+
+**Meetings.** Check the [sigstore community calendar](https://calendar.google.com/calendar/u/0?cid=ZnE0a2dvbTJjZTQzaG5jbmJjZmphMmNrMjBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for meeting invitations/times (see [community](https://github.com/sigstore/community) repository for more). We recognize that various constraints (time zones, connectivity, privacy concerns) mean that meetings aren't a great way for everybody to contribute. We strive to make important decisions asynchronously, via design documents and GitHub issues. At the same time, synchronous meetings can be really useful for hashing out complex issues quickly. We'll record these meetings (links should be in the notes docs).
+
+- sig-clients meeting (monthly; [notes](https://docs.google.com/document/d/1PNbBZSG3QC8hWVYBx6YDppaXwmSLDfx7t66ECaGa8y4/edit#heading=h.hb09y2f8i7to))
+- Sigstore Java (weekly; [notes](https://docs.google.com/document/d/1R7mL-IUrc2Z_LuOIvwDWshVuPQS_2VNE_cIQx4Oy5zw/edit))
+- Sigstore Golang (biweekly; [notes](https://docs.google.com/document/d/1EcJIhqSS9E86cHAQXaXiu2_r1s0kNbHz4uLLwwGo-vw/edit#heading=h.td0phy2bwk06))
+
+**Slack.** We also communicate on [Slack](https://links.sigstore.dev/slack-invite). Channels that might be of interest include: `#clients`, `#java`, `#ruby-gems`, `#sigstore-rust`, `#sigstore-go`, `#cosign`.
+
+## Governence
+
+We're still sorting this out for SIGs. Come back soon!
+
+## Roadmap
+
+You'll need to join [sigstore-dev@googlegroups.com](https://groups.google.com/g/sigstore-dev) for access to many of these (to prevent spam).
+
+Last updated: 2023-04-17. See also [project-wide roadmap](https://github.com/sigstore/community/blob/main/ROADMAP.md) (possibly out-of-date).
+
+*This currently represents @znewman01's brain dump; at the next client meeting, we'll work through this.*
+
+**Short-term (months).**
+
+Goal: focusing on Cosign in the near-term as the most widely-used client. If
+other clients like what we're doing there, they can emulate it. Document
+expectations for clients.
```

Comment:
Just wanted to +1 this for immediate focus for Cosign.

I think it'd also be good to include tasks for the other client languages too

Reply:
Mention that the goal is **not** to have each language client being complete replacement for the cosing cli too? I.e the language clients should focus on implementing support for signing and verification for ecosystem adoption. Any needed cli functionality to fully test the client is still encouraged.

Reply:
> I think it'd also be good to include tasks for the other client languages too

Added a generic "support other languages" item with a TODO to solicit per-language items at the next meeting.

> Mention that the goal is not to have each language client being complete replacement for the cosing cli too? 

Added a clause about "in-scope for each client".